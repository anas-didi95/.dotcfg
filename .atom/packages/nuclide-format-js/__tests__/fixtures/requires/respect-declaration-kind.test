const three = require('three');
let two = require('two');
var one = require('one');

one(two(three));
