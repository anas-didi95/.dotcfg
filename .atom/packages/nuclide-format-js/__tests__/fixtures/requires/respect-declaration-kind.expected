var one = require('one');
const three = require('three');
let two = require('two');

one(two(three));
